src/_rules/color.js

```diff
@@ -1,3 +1,15 @@
 export const opacity = [
   [/^opacity-?(\d+)$/, ([, d], { theme }) => ({ opacity: theme.opacity[d] }), { autocomplete: 'opacity-${opacity}' }],
```

Comment:
Remove optional thingy, code does not cover for anything else

```suggestion
  [/^opacity-(\d+)$/, ([, d], { theme }) => ({ opacity: theme.opacity[d] }), { autocomplete: 'opacity-${opacity}' }],
```

Reply:
Ah true, nice catch